spec/board_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../lib/board.rb'
+
+describe Board do
+  describe '#integer_count' do
+    it 'returns true if return value is an integer or nil' do
+      board = Board.new
+      count = board.integer_count
+      expect(count).to be_a Integer || Nil
+    end
+  end
+
+  describe '#update_board' do
+    it 'should return true if board is updated' do
+      curr_board = Board.new
+      player = Player.new('chris', 'O')
+      player_move = 5
+      curr_board.update_board(player_move, player)
+      expect(curr_board.board[4]).to eql('O')
+    end
```

Comment:
See the previous comment.

Also, this test is about `'#update_board'` why not test directly what does? 🤔 
e.g. => "it returns the player symbol when a board is updated"

Reply:
Hello @bolah2009.
Thanks for the review.
However our update board method doesn't actually return a player symbol but rather updates our board with that player symbol.
We have modified our test to check both for when the board is updated successfully, and when it doesn't.